### PR TITLE
Add backup `ExplicitSequenceFilenames` to update rules

### PR DESCRIPTION
**Pull request by PunkPun**
Adds https://github.com/OpenRA/OpenRA/pull/20580/commits/6c09805c656f57f8399ee274d9d5963287f69154 to update rules

Adresses https://github.com/OpenRA/OpenRA/pull/20580#issuecomment-1365073895

Due to the miniyaml refactor this will have issues when being merged into prep

### CODE REVIEW

**Comment by PunkPun** [comment]
I tested with https://resource.openra.net/maps/53325/

File: OpenRA.Mods.Common/UpdateRules/Rules/20230225/ExplicitSequenceFilenames.cs [hunk 1]
```diff
@@ -92,19 +92,88 @@ public override IEnumerable<string> BeforeUpdate(ModData modData)
 					defaultSpriteExtension = defaultSpriteExtensionNode.Value.Value;
 				}
 
-				var tilesetExtensionsNode = spriteSequenceFormatNode.LastChildMatching("TilesetExtensions");
+				var fromBackup = false;
+
+				var tilesetExtensionsNode = spriteSequenceFormatNode.LastChildMatching("TilesetExtensions")?.Value.Nodes;
```
Comment (penev92) [inline OpenRA.Mods.Common/UpdateRules/Rules/20230225/ExplicitSequenceFilenames.cs]: `?.Value?.Nodes`

Comment (PunkPun) [reply OpenRA.Mods.Common/UpdateRules/Rules/20230225/ExplicitSequenceFilenames.cs]: sure

**Review by penev92 — APPROVED** [review]

**Comment by penev92** [comment]
[Release changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/642e3d80d340eb914aee30615d0f832e5a80de06...a4e993812778edb6624e7a8a489818480e8bcf12)

[commit in prep](https://github.com/OpenRA/OpenRA/commit/cbf4207d22824d76e320fd0b1d00747ff8d7c9e8)

**Pull request merged**